pl/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Elixir School
+lang: pl
+---
+
+[![License](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
+
+Lekcje programowania w języku Elixir, inspirowane lekcjami Twittera [Scala School](http://twitter.github.io/scala_school/). 
+
+Dostepne w [Việt ngữ][vi], [汉语][cn], [Español][es], [日本語][jp] oraz [Português][pt].
```

Comment:
@kdebowski we've decided to remove these lines from all pages other than English.  Could you remove 11-17?